Tasks/AzureCLIV2/tsconfig.dev.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't want to add all these all at once, but once someone attempts to debug the task in which this file does not exist, let that person just add the file.

Reply:
I'm fine either way :).  It might work better from a generation perspective to add all-at-once.  Whatever is less friction for the change

Reply:
Adding them one by one would also let us easily track how widely it has been used to date; also it leaves more room for changing the approach if needed.

Last but not least; perhaps attaching these sources and source maps to the production tasks is not a bad idea and this would not be needed? WDYT?